tron/serialize/runstate/shelvestore.py

```diff
@@ -42,7 +42,7 @@ def build_key(self, type, iden):
 
     def save(self, key_value_pairs):
         for key, state_data in key_value_pairs:
-            self.shelve[key.key] = state_data
+            self.shelve[str(key.key)] = state_data
```

Comment:
This could have ramifications on our production clusters, nod?
Are any of our keys non-strings?

Reply:
i've checked - top level keys are always byte strings

i think we will get rid of shelve to store tron state anyway, but for now this should be safe